metadata.rb

```diff
@@ -2,7 +2,7 @@
 maintainer 'The Habitat Maintainers'
 maintainer_email 'humans@habitat.sh'
 license 'apache2'
-version '0.10.1'
+version '0.10.2'
```

Comment:
It's probably fine. 

![image](https://cloud.githubusercontent.com/assets/12483/22897929/cb239b22-f1e2-11e6-9e21-c4f629c10618.png)